storage/migrations/14_runtime_account_stats.up.sql

```diff
@@ -0,0 +1,37 @@
+BEGIN;
+
+CREATE TABLE stats.runtime_accounts
+(
+    runtime runtime NOT NULL,
+    address oasis_addr NOT NULL,
+    PRIMARY KEY (runtime, address),
+
+    num_txs UINT63 NOT NULL DEFAULT 0
+);
+
+-- Backfill stats.runtime_accounts
+INSERT INTO stats.runtime_accounts (runtime, address, num_txs)
+    SELECT runtime, account_address, COUNT(*) 
+    FROM chain.runtime_related_transactions
+    GROUP BY (runtime, account_address);
+
+CREATE OR REPLACE FUNCTION calculate_runtime_account_num_txs() RETURNS TRIGGER
```

Comment:
I thought we decided to avoid triggers? Let's move this into Go, please. We only perform the corresponding `INSERT INTO chain.runtime_related_transactions` in [one spot](https://github.com/oasisprotocol/nexus/blob/087ac66d6df6d1f9783b042057f0a7eddd57998d/analyzer/runtime/runtime.go#L180), so Go should be roughly the same amount of code.

Using just one mechanism makes code reading, debugging, and maintainance easier.
For example: For fast sync, we will need to disable certain triggers. It's easier to do that either in just Go or just in postgres.
Or: To understand how a table is updated, one has to code-spelunk (grep or otherwise). If all updates follow the same pattern, that's easier to do.

Reply:
Sounds good - updated